### PR TITLE
ref bug#132019 Fix time request of LE (M)

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/132019
- Fix time request of LE (M)
- Effect : List request personal

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java [hunk 1]
```diff
@@ -142,7 +144,7 @@ void setItemClickListener(OnRecyclerViewItemClickListener<Object> itemClickListe
         mItemClickListener = itemClickListener;
     }
 
-    private static class RequestOverTimeViewHolder extends RecyclerView.ViewHolder {
+    private class RequestOverTimeViewHolder extends RecyclerView.ViewHolder {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: WARNING: Missing a Javadoc comment.

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java [hunk 2]
```diff
@@ -155,12 +157,13 @@ void setItemClickListener(OnRecyclerViewItemClickListener<Object> itemClickListe
...
             mBinding.setViewModel(
-                    new ItemListRequestViewModel(requestOverTime, mItemClickListener, user));
+                    new ItemListRequestViewModel(mContext, requestOverTime, mItemClickListener,
+                            user));
             mBinding.executePendingBindings();
         }
     }
 
-    private static class RequestOffViewHolder extends RecyclerView.ViewHolder {
+    private class RequestOffViewHolder extends RecyclerView.ViewHolder {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: WARNING: Missing a Javadoc comment.

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java [hunk 3]
```diff
@@ -173,12 +176,12 @@ void bind(RequestOverTime requestOverTime, User user) {
...
         void bind(OffRequest requestOff, User user) {
             mBinding.setViewModel(
-                    new ItemListRequestViewModel(requestOff, mItemClickListener, user));
+                    new ItemListRequestViewModel(mContext, requestOff, mItemClickListener, user));
             mBinding.executePendingBindings();
         }
     }
 
-    private static class RequestLeaveViewHolder extends RecyclerView.ViewHolder {
+    private class RequestLeaveViewHolder extends RecyclerView.ViewHolder {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: WARNING: Missing a Javadoc comment.

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java [hunk 4]
```diff
@@ -25,6 +25,7 @@
 
 public class ListRequestAdapter extends BaseRecyclerViewAdapter<RecyclerView.ViewHolder> {
 
+    private Context mContext;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: pls remove this variable, U can use ``getContext().getApplicationContext()``

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java (hunk 1 above)
Comment (daolq3012) [reply app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: pls keep old code and pass context into constructor method of ViewHolder

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java (hunk 2 above)
Comment (daolq3012) [reply app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: like above

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java (hunk 3 above)
Comment (daolq3012) [reply app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestAdapter.java]: like above

**Pull request merged**